iam_robolib/src/feedback_controller/force_axis_impedence_feedback_controller.cpp

```
@@ -50,10 +52,15 @@ void ForceAxisImpedenceFeedbackController::get_next_step() {
 
 void ForceAxisImpedenceFeedbackController::get_next_step(const franka::RobotState &robot_state,
                                              TrajectoryGenerator *traj_generator) {
-    
+  
+  ImpulseTrajectoryGenerator* impulse_trajectory_generator = dynamic_cast<ImpulseTrajectoryGenerator*>(traj_generator);
+
+  if(impulse_trajectory_generator == nullptr) {
+    throw 333;
```

same here let's raise a valid exception here.